server/src/http/unleash_client.rs

```diff
@@ -1210,18 +1219,15 @@ mod tests {
             pkcs12_passphrase: None,
             pem_cert_file: Some(cert.into()),
         };
-        let client = new_reqwest_client(
-            false,
-            Some(identity),
-            None,
-            Duration::seconds(5),
-            Duration::seconds(5),
-            ClientMetaInformation {
+        let client = new_reqwest_client(HttpClientArgs {
+            client_identity: Some(identity),
+            client_meta_information: ClientMetaInformation {
                 app_name: "test-client".into(),
-                instance_id: "test-pkcs8".into(),
-                connection_id: "test-pkcs8".into(),
+                instance_id: "test-pem".into(),
+                connection_id: "test-pem".into(),
```

Comment:
Nice catch!

Reply:
Ooo, wish I could take credit. Co-pilot did this and I thought "huh, it's smarter than we are" so I kept it :laughing: